core/pom.xml

```diff
@@ -218,6 +218,16 @@
                 </executions>
             </plugin>
             -->
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Could we avoid the copypasta by activating this in the parent with a profile?  The profile could be active by default and suppressed by a `.javac-no-parameters` flag file or the like.